jina/serve/streamer.py

```diff
@@ -189,8 +189,6 @@ async def close(self):
         await self._streamer.wait_floating_requests_end()
         await self._connection_pool.close()
 
-    Call = stream
```

Comment:
I think for (internal) compatibility there's no harm in keeping the streamer bindable to the grpc server

Reply:
yes, no harm at all u are right